src/microbe_stage/Microbe.tscn

```diff
@@ -21,8 +21,14 @@ shader_param/tint = Plane( 1, 1, 1, 1 )
 shader_param/albedoTexture = ExtResource( 2 )
 shader_param/damagedTexture = ExtResource( 3 )
 
-[sub_resource type="SphereShape" id=2]
-resource_local_to_scene = true
+[sub_resource type="SphereShape" id=7]
+
+[sub_resource type="SpatialMaterial" id=5]
+render_priority = 2
```

Comment:
Does this need to be a scene specific resource?

Reply:
These are previous pseudopod related debug nodes, which will potentially be used again later. And yes I don't think it's worth it making these non-scene specific.

Reply:
> And yes I don't think it's worth it making these non-scene specific.

That wording is a bit confusing... Do you think these should be made scene specific or not? If there isn't a foreseen need to edit their properties separately, then I'm fine with leaving them as they are now.

Reply:
They should be scene specific.

Reply:
Done I think.

Reply:
I see `resource_local_to_scene = true` now. Though is the sphere mesh also edited? That also got the scene local property. If the sphere mesh *isn't* modified (like the size, position etc. doesn't matter) that should not be scene specific to not create unnecessary object instances.

Reply:
Should be fixed.